iceoryx_hoofs/include/iceoryx_hoofs/error_handling/error_handling.hpp

```diff
@@ -17,20 +17,26 @@
 #ifndef IOX_HOOFS_ERROR_HANDLING_ERROR_HANDLING_HPP
 #define IOX_HOOFS_ERROR_HANDLING_ERROR_HANDLING_HPP
 
+/// @todo #1099 rename this file to error_handler.hpp
```

Comment:
This todo is not listed in the issue, please add it since it may comes with some bigger code refactoring.